extensions/wikia/Discussions/maintenance/ForumDumper.php

```diff
@@ -111,6 +111,13 @@ public function getPages() {
 			->WHERE( 'page_namespace' )->IN( self::FORUM_NAMEPSACES )
 			->ORDER_BY( 'idx' )
 			->runLoop( $dbh, function ( &$pages, $row ) use ( &$display_order ) {
+				// A few of these properties were removed and do not appear on some wikis
+				foreach ( [ 'sticky', 'locked', 'protected' ] as $prop ) {
+					if ( !property_exists( $row, $prop ) ) {
+						$row->$prop = 0;
+					}
+				}
+
```

Comment:
@TK-999, thanks for the update!  I've seen some still in DEV so its good to know I don't need to worry about them in production.